core/src/main/java/tc/oc/pgm/observers/tools/FlySpeedTool.java

```diff
@@ -17,6 +17,12 @@
 
   private static String TRANSLATION_KEY = "setting.flyspeed.";
 
+  InventoryMenu in;
+
+  public FlySpeedTool(InventoryMenu in) {
+    this.in = in;
```

Comment:
```suggestion
  InventoryMenu menu;

  public FlySpeedTool(InventoryMenu menu) {
    this.menu = menu;
```

Reply:
Wops this is just code I forgot to remove